lib/parse.js

```diff
@@ -0,0 +1,12 @@
+ /**
+ * Get git url of current project.
+ * @function getRepoUrl
+ * @returns {String}
+ */
+const get = require('./get');
+const lget = require('lodash.get');
+function getRepoUrl () {
```

Comment:
Since repo url can be more than one and you only return the origin specificly here, i would change the function name to getRepoOriginUrl.
Actually i would create another function for this.
And i would change this one to get the remote name
something like this:
```js
function getRepoUrl (remoteName) {
  return lget(get.sync(),`remote.${remoteName}.url`);
}
```
Than in the getRepoOriginUrl i would just call it with origin as arg
(Don't forget to update the readme)